core/tx_pool.go

```diff
@@ -540,13 +555,17 @@ func (pool *TxPool) validateTx(tx *types.Transaction, local bool) error {
 	if pool.currentMaxGas < tx.Gas() {
 		return ErrGasLimit
 	}
+	// Ensure feeCap is less than or equal to tip.
```

Comment:
At L540, Please also reject the 1559 txs if the fork is not happened yet.
Although during the transaction processing, the 1559 transactions will be filtered out, but would be nice to reject it in the first place.

Reply:
done